Gruntfile.js

```diff
@@ -145,6 +146,7 @@ grunt.initConfig( {
 			"test/main/modules",
 			"test/main/deepEqual",
 			"test/main/stack",
+            "test/onerror/onerror",
```

Comment:
Oops, need to tabify this line. I can't wait until we can get ESLint indent working...